templates/includes/sidebar.html

```diff
@@ -1,4 +1,8 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
Ensure that the `driver-detail` URL pattern expects a `pk` parameter that corresponds to the `user.id`. If `user.id` does not match the expected parameter for the `driver-detail` view, this could lead to a `NoReverseMatch` error. Verify that the `pk` is the correct parameter for this URL.